.github/workflows/refresh_looker_metrics.yml

```diff
@@ -19,26 +19,20 @@ name: Refresh Looker Performance Metrics
 
 on:
   workflow_dispatch:
-    inputs:
-      READ_ONLY:
-        description: 'Run in read-only mode'
-        required: false
-        default: 'true'
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   LOOKERSDK_BASE_URL: ${{ secrets.LOOKERSDK_BASE_URL }}
   LOOKERSDK_CLIENT_ID: ${{ secrets.LOOKERSDK_CLIENT_ID }}
   LOOKERSDK_CLIENT_SECRET: ${{ secrets.LOOKERSDK_CLIENT_SECRET }}
   GCS_BUCKET: 'public_looker_explores_us_a3853f40'
-  READ_ONLY: ${{ inputs.READ_ONLY }}
```

Comment:
Is this related to the vllm pieces? Rather than removing it entirely, we could also just flip the default to false

Reply:
I found that this parameter was added by mistake and never used